services/report/report_processor.py

```diff
@@ -55,6 +55,26 @@
     buckets=[0.05, 0.1, 0.5, 1, 2, 5, 7.5, 10, 15, 20, 30, 60, 120, 180, 300, 600, 900],
 )
 
+KiB = 1024
+MiB = KiB * KiB
+RAW_REPORT_SIZE = Histogram(
+    "worker_services_report_raw_report_size",
+    "Size (in bytes) of a raw report",
+    ["processor"],
+    buckets=[
+        10 * KiB,
+        100 * KiB,
+        200 * KiB,
+        500 * KiB,
+        1 * MiB,
+        2 * MiB,
+        5 * MiB,
+        10 * MiB,
+        20 * MiB,
+        50 * MiB,
```

Comment:
can you add some higher buckets? maybe 200, 500, 1000, 2000 or something? measurements higher than the largest bucket are effectively clamped. if your 75th percentile is larger than your max bucket, a p75 graph will lie to you and say the max bucket size is your p75

we don't know the distribution of these sizes so it maybe won't be an issue, but i don't think we currently deal with so much prometheus data that the extra buckets will kill us. we could also thin out some of the lower bucket granularities if we are concerned about that, since for our purposes there is not much difference between 10kb and 100kb or 1mb and 2mb

(i like the `KiB` and `MiB` constants + thanks for a good metric description!)